pylint/pyreverse/mermaidjs_printer.py

```diff
@@ -24,6 +24,7 @@ class MermaidJSPrinter(Printer):
         EdgeType.ASSOCIATION: "--*",
         EdgeType.AGGREGATION: "--o",
         EdgeType.USES: "-->",
+        EdgeType.TYPE_DEPENDENCY: "-.->",
```

Comment:
this is the actual wrong line

Reply:
Thanks! Would you be willing to provide a PR?

Reply:
yep... just did!
#9031